src/hooks/useEditorEffect.ts

```diff
@@ -34,7 +34,9 @@ export function useEditorEffect(
   // be defined inline and run on every re-render.
   useLayoutGroupEffect(
     () => {
-      if (editorView) effect(editorView);
+      if (editorView) {
+        return effect(editorView);
+      }
     },
```

Comment:
When we added this check for `editorView`, we unintentionally stopped returning the result of calling `effect`. This meant that we were no longer executing cleanup functions ever!

Reply:
#104